build_bundle_image.sh

```diff
@@ -271,13 +276,31 @@ function push_docker_image {
 function set_parent_image {
 	if (echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | grep -q "q")
 	then
-		return
-	fi
+		if [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1)" -gt 2024 ]]
+		then
+			return
+		fi
 
-	if [ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION%-*}" | cut -f1,2,3 -d'.' | cut -f1 -d '-' | sed 's/\.//g' )" -le 7310 ]
+		if [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1)" -eq 2024 ]] &&
+		   [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 2 | tr -d q)" -ge 3 ]]
+		then
+			return
+		fi
+
+		sed -i 's/liferay\/jdk21:latest AS liferay-jdk21/liferay\/jdk11:latest AS liferay-jdk11/g' "${TEMP_DIR}"/Dockerfile
+		sed -i 's/FROM liferay-jdk21/FROM liferay-jdk11/g' "${TEMP_DIR}"/Dockerfile
+	elif [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1,2)" == 7.4* ]] ||
+	     [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1,2)" == 7.3* ]]
+	then
+		sed -i 's/liferay\/jdk21:latest AS liferay-jdk21/liferay\/jdk11:latest AS liferay-jdk11/g' "${TEMP_DIR}"/Dockerfile
+		sed -i 's/FROM liferay-jdk21/FROM liferay-jdk11/g' "${TEMP_DIR}"/Dockerfile
+	elif [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION%-*}" | cut -f1,2,3 -d'.' | cut -f1 -d '-' | sed 's/\.//g' )" -le 7310 ]]
```

Comment:
@natocesarrego, those cut commands seem unnecessary, because  `"$(echo "${LIFERAY_DOCKER_RELEASE_VERSION%-*}"` already remove the `-` and all `.` will be removed with `sed 's/\.//g'`

Reply:
Considering an old version, like `7.2.10.8`, I understand that at least the first `cut` is necessary, but I'll double check. :wink: Thanks for raising this flag. :slightly_smiling_face: